### PR TITLE
Adjust the query to keep the destination rather than replace.

### PR DESCRIPTION
**Pull request by calebbrown**
#1000

### CODE REVIEW

File: scripts/bq_load.sh [hunk 1]
```diff
@@ -62,7 +62,20 @@ for bucket_prefix in `gsutil ls "$RESULT_BUCKET"`; do
...
+# If the table does not exist it will be created. Keeping the table ensures
+# that breaking schema changes are not accidentally propogated to the
+# destination table.
+#
+# A transaction is used to keep the update atomic.
+query="
+CREATE TABLE IF NOT EXISTS \`$PROJECT_ID.$DEST_DATASET.$DEST_TABLE\` LIKE \`$PROJECT_ID.$LOAD_DATASET.$table_name\`
+    PARTITION BY TIMESTAMP_TRUNC(CreatedTimestamp, DAY) OPTIONS(expiration_timestamp=NULL);
+BEGIN TRANSACTION;
+TRUNCATE TABLE \`$PROJECT_ID.$DEST_DATASET.$DEST_TABLE\`;
```
Comment (maxfisher-g) [inline scripts/bq_load.sh:76]: Is the point that the transaction (including truncation) will fail due to the insert if there are any breaking schema changes?

May be worth adding this explicitly on line 71

File: scripts/bq_load.sh [hunk 2]
```diff
@@ -62,7 +62,20 @@ for bucket_prefix in `gsutil ls "$RESULT_BUCKET"`; do
     union="$union$subquery"
 done
 
-query="CREATE OR REPLACE TABLE \`$PROJECT_ID.$DEST_DATASET.$DEST_TABLE\` LIKE \`$PROJECT_ID.$LOAD_DATASET.$table_name\` PARTITION BY TIMESTAMP_TRUNC(CreatedTimestamp, DAY) OPTIONS(expiration_timestamp=NULL) AS $union;"
+# Query to populate the destination table.
+#
+# If the table does not exist it will be created. Keeping the table ensures
+# that breaking schema changes are not accidentally propogated to the
```
Comment (maxfisher-g) [inline scripts/bq_load.sh:68]: typo: propogated -> propagated

Comment (calebbrown) [reply scripts/bq_load.sh]: Done

File: scripts/bq_load.sh (hunk 1 above)
Comment (calebbrown) [reply scripts/bq_load.sh:77]: Done!

**Review by maxfisher-g — APPROVED** [review]

**Pull request merged**